.github/workflows/matrix-extensions.json

```diff
@@ -1,79 +1,110 @@
             "wasi_logging",
-            "wasmedge_process",
-            "wasmedge_tensorflow",
-            "wasmedge_tensorflowlite",
+            "wasi_nn-ggml",
+            "wasi_nn-tensorflowlite",
+            "wasi_nn-whisper",
+            "wasmedge_ffmpeg",
             "wasmedge_image",
             "wasmedge_opencvmini",
-            "wasmedge_ffmpeg"
+            "wasmedge_process",
+            "wasmedge_stablediffusion",
+            "wasmedge_tensorflow",
+            "wasmedge_tensorflowlite"
         ],
         "manylinux_2_28_x86_64": [
-            "wasi_nn-tensorflowlite",
-            "wasi_nn-wasi_nn-ggml",
-            "wasi_nn-wasi_nn-pytorch",
-            "wasm_bpf",
             "wasi_crypto",
             "wasi_logging",
+            "wasi_nn-ggml",
+            "wasi_nn-pytorch",
+            "wasi_nn-tensorflowlite",
+            "wasi_nn-whisper",
+            "wasm_bpf",
+            "wasmedge_ffmpeg",
+            "wasmedge_image",
+            "wasmedge_opencvmini",
             "wasmedge_process",
+            "wasmedge_stablediffusion",
             "wasmedge_tensorflow",
             "wasmedge_tensorflowlite",
-            "wasmedge_image",
-            "wasmedge_opencvmini",
-            "wasmedge_ffmpeg"
+            "wasmedge_zlib"
         ],
         "manylinux_2_28_aarch64": [
-            "wasi_nn-tensorflowlite",
-            "wasi_nn-wasi_nn-ggml",
             "wasi_crypto",
             "wasi_logging",
+            "wasi_nn-ggml",
+            "wasi_nn-tensorflowlite",
+            "wasi_nn-whisper",
+            "wasmedge_ffmpeg",
+            "wasmedge_image",
+            "wasmedge_opencvmini",
             "wasmedge_process",
+            "wasmedge_stablediffusion",
             "wasmedge_tensorflow",
             "wasmedge_tensorflowlite",
-            "wasmedge_image",
-            "wasmedge_opencvmini",
-            "wasmedge_ffmpeg"
+            "wasmedge_zlib"
         ]
     },
     "detail": {
-        "wasi_nn-tensorflowlite": {
-            "plugin": "wasi_nn-tensorflowlite",
+        "wasi_crypto": {
+            "plugin": "wasi_crypto",
+            "bin": "libwasmedgePluginWasiCrypto.so",
+            "dir": "wasi_crypto",
+            "testBin": "wasiCryptoTests",
+            "options": "-DWASMEDGE_PLUGIN_WASI_CRYPTO=ON"
+        },
+        "wasi_logging": {
+            "plugin": "wasi_logging",
+            "bin": "libwasmedgePluginWasiLogging.so",
+            "dir": "wasi_logging",
+            "testBin": "wasiLoggingTests",
+            "options": "-DWASMEDGE_PLUGIN_WASI_LOGGING=ON"
+        },
+        "wasi_nn-ggml": {
+            "plugin": "wasi_nn-ggml",
             "bin": "libwasmedgePluginWasiNN.so",
             "dir": "wasi_nn",
             "testBin": "wasiNNTests",
-            "options": "-DWASMEDGE_PLUGIN_WASI_NN_BACKEND=TensorFlowLite"
+            "options": "-DWASMEDGE_PLUGIN_WASI_NN_BACKEND=GGML -DWASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_BLAS=OFF"
```

Comment:
We don't need `-DWASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_BLAS=OFF` now. The default value is changed to `OFF`.